recipes/ucsc-parafetch/meta.yaml

```diff
@@ -1,57 +1,64 @@
 {% set package = "ucsc-parafetch" %}
 {% set program = "paraFetch" %}
-{% set version = "469" %}
-{% set sha256 = "3f95b26ccca854c965c05773481c269f2a0e6fb1906ae8a7fe2437c6ad543963" %}
+{% set version = "482" %}
+{% set sha256 = "745d0f0a980e9f863373b0b0a6ad7961a1d0f53a9d7c446e92fc28fc4909431a" %}
 
 package:
   name: "{{ package }}"
   version: "{{ version }}"
 
 source:
-  url: "http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v{{ version }}.src.tgz"
+  url: "https://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v{{ version }}.src.tgz"
   sha256: "{{ sha256 }}"
   patches:
-    - "include.patch"
+    - 0001-include.patch
+    - 0002-include.linux.patch  # [linux]
+    - 0003-common.mk.patch
+    - 0004-straw.patch
 
 build:
-  skip: True  # [osx]
-  number: 1
+  number: 0
   run_exports:
     - {{ pin_subpackage(package, max_pin=None) }}
 
 requirements:
   build:
     - make
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
     - libpng
     - libuuid
-    - mysql-connector-c
+    - mysql-connector-c # [not arm64]
     - libopenssl-static
+    - clangdev  # [osx]
+    - rsync  # [osx]
+    - libiconv
+    - bzip2
+    - liblzma-devel
     - zlib
-
   run:
     - libpng
```

Comment:
libpng and probably also the other libs here do not need explicit run-exports here because they define them themself ... see here: https://github.com/conda-forge/libpng-feedstock/blob/main/recipe/meta.yaml#L19

So we could remove all those unnecessary run-deps